src/routeHelpers.jsx

```diff
@@ -155,3 +155,13 @@ export function isSubjectPagePath(pathname) {
   }
   return false;
 }
+
+export function toProgramme(programmePath) {
+  return `${PROGRAMME_PATH}/${programmePath}`;
+}
+
+export function toProgrammeSubject(programmePath, subjectId, filterIds) {
+  const baseUrl = `${toProgramme(programmePath)}${toSubject(subjectId)}/`;
```

Comment:
@kqpetter Det er denne linja eg synes blir feil. Den legger på programme-pathen til subject-pathen. Så når du klikker på en av lenkene herfra blir den slik: /utdanning/bygg-og-anleggsteknikk/subjects/subject:11/?filters=urn:filter:0d67724e-d9fa-4365-9839-4cc91c012855.  Når du seinere klikker deg videre så strippes første delen av urlen ned til /subjects/*. 
```suggestion
  const baseUrl = `${toSubject(subjectId)}/`;
```

Reply:
@gunnarvelle - ser ut til å fungere greit det. Vet ikke om tanken/hensikten her var at subject* sidene skulle eksistere under slugs med tanke på SEO, men det er kanskje ikke korrekt?

Reply:
Det er mulig men då må vil i tilfelle endre en del plasser for å passe på at korrekt path tas vare på og ikkje strippes vekk. Det enkleste no er å gjøre dette i første omgang for å få dette ut og så kan vi heller legge det tilbake før skulestart.